src/components/CheckboxInput/CheckboxInput.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { v4 as uuid } from 'uuid';
+import './CheckboxInput.scss';
+
+const CheckboxInput = ({
+  id,
+  className,
+  isSelected,
+  isDisabled,
+  onChange,
+  children,
+}) => {
+  const [selected, setSelected] = useState(isSelected || false);
+  const inputId = id || uuid();
+
+  useEffect(() => {
```

Comment:
Hey @JCarmine just a heads up, I ran this locally but seems the effect is creating an infinite loop. Let's chat about this tomorrow. I feel we can probably get rid of this altogether, but want to hear what you think.

Reply:
It's already gone. I'll push soon.